docs/3.x/tech/tutorials/exporting-lists.adoc

```diff
@@ -0,0 +1,35 @@
+---
+layout: doc
+title: Exporting Lists and Configurations
+description: Learn how to export your lists and Configurations in CleanSpeak
+---
+
+
+== Exporting Lists
+Note: Making changes to your lists after exporting will cause issues when importing.
+
+
+You may be asked by our support team to export your lists and configurations so that we can test it on our end.
+Maybe you're moving lists between systems, or you're doing an upgrade and you want to backup your CleanSpeak DB.
+Here we will show you how to quickly and easily export your lists and configurations in CleanSpeak.
+
+
+From the moderator dashboard you will click on the filter drop down found in the left navigation.
```

Comment:
This is not the correct name in the left nav. 

> ... click on the `Filters > Lists` from the left navigation. 

I think we have some style for bread crumb instructions, see FusionAuth release notes. I think it is something like `[breadcrumb]#Filters -> Lists#`.